src/24-uncensorable-wikipedia/index.md

```diff
@@ -0,0 +1,196 @@
+
+So far, we have achieved part of this goal: we have static snapshots of all of Wikipedia on IPFS. This is alrady a huge result that will help people access, keep, archive, cite, and distribute lots of content. In particular, we hope that this distribution helps people in Turkey, who find themselves in a tough situation. We are still working out a process to continue updating these snapshots, we hope to have someone at Wikimedia in the loop as they are the authoritative source of the content. **If you could help with this, please get in touch with us at wikipedia-project@ipfs.io.**
+
+### (Goal 2) Fully Read-Write Wikipedia on IPFS
+
+The long term goal is to get the full-fledged read-write Wikipedia to work on top of IPFS. This is much more difficult because for a read-write application like Wikipedia to leverage the distributed nature of IPFS, we need to change the how the applications write data. A read-write wikipedia on IPFS would allow completely decentralized and extremely difficult to censor operation. In addition to all the benefits of the static version above, the users of a read-write Wikipedia on IPFS could write content from anywhere and publish it, even without being directly connected to any wikipedia.org servers. There would be automatic version control and version history archiving. We could allow people to view, edit, and publish in completely encrypted contexts, which is important to people in highly repressive regions of the world.
+
+A full read-write version (2) would require a strong collaboration with Wikipedia.org itself, and finishing work on important dynamic content challenges -- we are working on all the technology (2) needs, but it's not ready for prime-time yet. We will update when it is.
+
+## Wikipedia on IPFS -- The Release Today
+
+Today, we are releasing the first full static snapshots on IPFS of all of https://en.wikipedia.org and https://tk.wikipedia.org. We describe how to access them, and how we did this below. These snapshots were taken on `2017-04-30`.
+
+The unique identifier (cryptographic hash) for the snapshot of Wikipedia from April 30th is:
+* English Wikipedia (30 April 2017): [en-wikipedia-hash](http://ipfs.io/ipfs/en-wikipedia-hash/out/A/Anasayfa.html)
+* Turkish Wikipedia (30 April 2017): [QmQP99yW82xNKPxXLroxj1rMYMGF6Grwjj2o4svsdmGh7S](http://ipfs.io/ipfs/QmQP99yW82xNKPxXLroxj1rMYMGF6Grwjj2o4svsdmGh7S/out/A/Anasayfa.html)  
+
+These links point to specific snapshots. They will always point to that specific day's snapshot. To find the most up-to-date snapshot of Wikipedia on IPFS you can use these IPNS links, which will resolve to the latest snapshot whenever we release new ones:
+* English Wikipedia (most recent snapshot): [en-wikipedia-ipns](http://ipfs.io/ipfs/en-wikipedia-ipns/out/A/Anasayfa.html)
```

Comment:
I think it would be best if the IPNS sites were simple JS redirects to IPFS based ones. It would improve speed of resolution after the first and would not relay on IPNS as much. In future we could have 'update notifications' in the IPFS based wiki.